roles/openstack-package/tasks/yum.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: "install {{ project_name }} package"
+  yum:
+    name: openstack-{{ project_name }}
+    state: present
+  register: project_package
+  until: project_package|succeeded
+  retries: 5
+
+- name: ensure project config directory exists
+  file: name="/etc/{{ project_name }}" state=directory
+  when: openstack_package.rootwrap|bool
+
+- name: stat rootwrap.d
```

Comment:
Hmmm, i think i need to rip out all the rootrwap stuff for the yum task. I don't think it's needed. So i will rip out these lines since we are not using a venv anyway and the end goal of the rootrwap logic is to setup a symlink in the venv